Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/run.ads

```diff
@@ -0,0 +1,9 @@
+
+with GL.Objects.Programs;
+
+with Window_Types;
+
+package Run is
+    procedure Main_Loop(Main_Window    : in out Window_Types.tWindow;
+                                         Rendering_Program   : GL.Objects.Programs.Program);
```

Comment:
colon misalignment. Also, why is this a package? Make it a procedure (see comment on `Render` above).